tgmc.dme

```diff
@@ -542,7 +542,6 @@
 #include "code\game\objects\items\books\book.dm"
 #include "code\game\objects\items\books\manuals.dm"
 #include "code\game\objects\items\circuitboards\AI_modules.dm"
-#include "code\game\objects\items\circuitboards\airlock.dm"

```

Comment:
accident i think

Reply:
Naw, consolidated the circuits cause they should be consolidated.